packages/mui-joy/src/Menu/Menu.tsx

```diff
@@ -103,9 +103,9 @@ const Menu = React.forwardRef(function Menu(inProps, ref: React.ForwardedRef<HTM
   const color = disablePortal ? getColor(inProps.color, colorProp) : colorProp;
 
   const handleOpenChange = React.useCallback(
-    (isOpen: boolean) => {
+    (isOpen: boolean, e?: React.MouseEvent<Element, Event>) => {
```

Comment:
```suggestion
    (isOpen: boolean, event?: React.MouseEvent<Element, Event>) => {
```

(see https://github.com/mui/material-ui/pull/37064)

Reply:
resolved in this commit https://github.com/mui/material-ui/pull/36917/commits/a6c11e01de9d805e467bafb413fd3bd5163f692b

>We can also open another PR to introduce a similar change to useSelect. @sai6855, would you be up for it?

sure I'll create similar PR for `useSelect`